src/WebApiToTypeScript/Endpoints/EndpointsService.cs

```diff
@@ -42,13 +49,13 @@ public void WriteEndpointClassToBlock(TypeScriptBlock endpointsBlock, WebApiCont
                 .AddAndUseBlock($"export {Config.NamespaceOrModuleName} {webApiController.Name}");
 
             TypeScriptBlock serviceBlock = null;
-            if (Config.GenerateService)
+            if (Config.GenerateService && !isMobile)
```

Comment:
Let's keep a separate MobileEndpointsService class and thus we can remove blocks like these. Please re-add the underscore lib removal though to EndpointsService ;)